app-text/mandoc/mandoc-1.14.3.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A suite of tools compiling mdoc from the OpenBSD project"
+DESCRIPTION="A suite of tools compiling mdoc, the roff macro language of choice \
+for BSD manual pages, and man, the predominant historical language for UNIX \
+manuals. It is small, ISO C, ISC-licensed, and quite fast. The main component \
+of the toolset is the mandoc utility program, based on the libmandoc validating \
+compiler, to format output for UTF-8 and ASCII UNIX terminals, HTML 5, \
+PostScript, and PDF."
+HOMEPAGE="http://mandoc.bsd.lv/"
+COPYRIGHT="1989-2017 mandoc Authors"
```

Comment:
Please copy the authors list in the license text here.

Reply:
The list is growing and changing after every release, that's why I kept them outside